app/hotels/src/filter/__tests__/FiltersReducer.test.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+
+import FiltersReducer, { InitialFiltersReducerState } from '../FiltersReducer';
+
+const type = 'filtersReducer/FILTER_CHANGED';
+
+describe('FiltersReducer', () => {
+  it('should return the initialState', () => {
+    // $FlowExpectedError: Need to test that it returns the inital state when called first time
```

Comment:
Just a small note: this comment is not that helpful because you already revealed your intentions in the test description (`describe` + `it`). It's more important to explain why are you actually suppressing this error. So for example: _"the second argument is intentionally not a valid action"_... :)

I will fix it.